src/Suggestions.js

```diff
@@ -75,7 +75,15 @@ class Suggestions extends Component {
     const { [this.props.labelField]: labelValue } = input;
 
     return {
-      __html: labelValue.replace(RegExp(escapedRegex, 'gi'), '<mark>$&</mark>'),
+      __html: labelValue.replace(RegExp(escapedRegex, 'gi'), (x) => {
+        const escapedInput = x
+          .replace(/&/g, '&amp;')
```

Comment:
@devinlundberg can you move the escape logic to the `utils.js` and add spec for util as well ?